client/src/components/navigation/Navigation.tsx

```diff
@@ -23,21 +23,18 @@ class Navigation extends React.Component<Props, {}> {
             </Menu.Item>
         </Link>);
 
-        const loginLink = !user ? (<a href="/auth/login"><Menu.Item>Sign in</Menu.Item></a>) : null;
+        const loginLink = !user ? (<Menu.Item href="/auth/login">Sign in</Menu.Item>) : null;
```

Comment:
do you need the `as={Link} to` syntax you use below?

Reply:
Maybe, you have to be logged in to see the frontend so it's a big edge case

Reply:
Update: the answer is no.  Menu.Item supports linking via an `href` attribute.  See https://react.semantic-ui.com/collections/menu/#content-link-item